dbx/src/main/java/dev/chrisbanes/insetter/InsetterBindingAdapters.java

```diff
@@ -102,6 +102,22 @@ public void onApplyInsets(
         });
   }
 
+  @BindingAdapter(
+            value = {
+                    "marginBottomForKeyboard",
```

Comment:
Can we rename this to `marginBottomForIme`? Since this is only needed for on screen keyboards, I think it's better to be clear